### PR TITLE
CVE fix

### PR DESCRIPTION
**Pull request by bowenlan-amzn**
Signed-off-by: bowenlan-amzn <bowenlan23@gmail.com>

*Description of changes:*
1. Fix several CVEs
2. Rename zip to kebab case

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
have the right to submit it under the open source license
indicated in the file; or

(b) The contribution is based upon previous work that, to the best
of my knowledge, is covered under an appropriate open source
license and I have the right under that license to submit that
work with modifications, whether created in whole or in part
by me, under the same open source license (unless I am
permitted to submit under a different license), as indicated
in the file; or

(c) The contribution was provided directly to me by some other
person who certified (a), (b) or (c) and I have not modified
it.

(d) I understand and agree that this project and the contribution
are public and that a record of the contribution (including all
personal information I submit with it, including my sign-off) is
maintained indefinitely and may be redistributed consistent with
this project or the open source license(s) involved.

### CODE REVIEW

**Comment by bowenlan-amzn** [comment]
Waiting for OpenSearch 1.0.0 distribution

File: .github/workflows/cypress-workflow.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
         with:
           repository: 'opensearch-project/OpenSearch'
           path: OpenSearch
-          ref: '1.0'
+          ref: '1.0.0-rc1'
```
Comment (thalurur) [inline .github/workflows/cypress-workflow.yml:32]: why is this needed?

File: .github/workflows/cypress-workflow.yml [hunk 2]
```diff
@@ -39,7 +39,7 @@ jobs:
         with:
           repository: 'opensearch-project/common-utils'
           path: common-utils
-          ref: 'main'
+          ref: '1.0.0.0-rc1'
```
Comment (thalurur) [inline .github/workflows/cypress-workflow.yml:42]: can we use `1.0` instead?

File: .github/workflows/cypress-workflow.yml [hunk 3]
```diff
@@ -49,7 +49,7 @@ jobs:
         with:
           repository: 'opensearch-project/job-scheduler'
           path: job-scheduler
-          ref: 'main'
+          ref: '1.0.0.0-rc1'
```
Comment (thalurur) [inline .github/workflows/cypress-workflow.yml:52]: same as above, can we use `1.0`?

Comment (bowenlan-amzn) [reply .github/workflows/cypress-workflow.yml]: changed

File: .github/workflows/cypress-workflow.yml (hunk 1 above)
Comment (bowenlan-amzn) [reply .github/workflows/cypress-workflow.yml]: changed

File: .github/workflows/cypress-workflow.yml (hunk 2 above)
Comment (bowenlan-amzn) [reply .github/workflows/cypress-workflow.yml]: changed

**Review by thalurur — APPROVED** [review]
